service/test/lib/integration/teamOpsTest.js

```diff
@@ -154,14 +154,52 @@ lab.experiment('TeamOps', () => {
     })
   })
 
+  // TODO: Needs review
```

Comment:
@p16 need to be checked against the changes made by @nherment. With the dynamic team id the `readTeam` wasn't returning any results so the tests data needs to be extended or check if ok to remain as it is relaying on external test data from db.